core/src/stored/backends/chunked_device.cc

```diff
@@ -681,8 +681,8 @@ ssize_t ChunkedDevice::ReadChunked(int, void* buffer, size_t count)
         && current_chunk_->end_offset >= (boffset_t)((offset_ + count) - 1)) {
       wanted_offset = (offset_ % current_chunk_->chunk_size);
 
-      bytes_left
-          = MIN((ssize_t)count, (current_chunk_->buflen - wanted_offset));
+      bytes_left = MIN((ssize_t)count,
+                       (ssize_t)(current_chunk_->buflen - wanted_offset));
```

Comment:
```suggestion
                       ((ssize_t)current_chunk_->buflen - wanted_offset));
```
This does two conversions otherwise.  Its probably better to just have one conversion

Reply:
Fixed. Should I fix the same issue here, too? https://github.com/bareos/bareos/blob/master/core/src/stored/backends/chunked_device.cc#L713

Reply:
Yes that would be great!

Reply:
I fixed multiple similar issues.